recipes/repl-builtin-modules/src/workflow.ts

```diff
@@ -0,0 +1,309 @@
+import { EOL } from "node:os";
+import { getNodeImportStatements } from "@nodejs/codemod-utils/ast-grep/import-statement";
+import { getNodeRequireCalls } from "@nodejs/codemod-utils/ast-grep/require-call";
+import { resolveBindingPath } from "@nodejs/codemod-utils/ast-grep/resolve-binding-path";
+import type { SgRoot, Edit, SgNode } from "@codemod.com/jssg-types/main";
+
+const containsBuiltinProperties = (text: string): boolean =>
+    text.includes("builtinModules") || text.includes("_builtinLibs");
+
+/**
+ * Transform function that converts deprecated repl.builtinModules and repl._builtinLibs
+ * to module.builtinModules API.
+ *
+ * See https://nodejs.org/api/deprecations.html#DEP0191
+ *
+ * Handles:
+ * 1. const repl = require('node:repl'); repl.builtinModules → const module = require('node:module'); module.builtinModules
+ * 2. const { builtinModules } = require('node:repl'); → const { builtinModules } = require('node:module');
+ * 3. const { builtinModules, foo } = require('node:repl'); → const { foo } = require('node:repl'); const { builtinModules } = require('node:module');
+ * 4. import { builtinModules } from 'node:repl'; → import { builtinModules } from 'node:module';
+ * 5. import { builtinModules, foo } from 'node:repl'; → import { foo } from 'node:repl'; import { builtinModules } from 'node:module';
+ * 6. const repl = require('node:repl'); repl._builtinLibs → const module = require('node:module'); module.builtinModules
+ * 7. const { _builtinLibs } = require('node:repl'); → const { builtinModules } = require('node:module');
+ * 8. import { _builtinLibs } from 'node:repl'; → import { builtinModules } from 'node:module';
+ */
+export default function transform(root: SgRoot): string | null {
+    const rootNode = root.root();
+    const edits: Edit[] = [];
+
+    const replaceStandaloneBuiltinLibsReferences = (): void => {
+        const standaloneReferences = rootNode.findAll({
+            rule: { pattern: "_builtinLibs" }
+        });
+
+        for (const ref of standaloneReferences) {
+            const parent = ref.parent();
+
+            if (
+                parent
+                && parent.kind() !== "object_pattern"
+                && parent.kind() !== "member_expression"
+                && parent.kind() !== "named_imports"
+            ) {
+                edits.push(ref.replace("builtinModules"));
+            }
+        }
+    };
+
+    const updateModuleSpecifier = (statement: SgNode): void => {
+        const moduleSpecifier = statement.find({ rule: { kind: "string" } });
+
+        if (moduleSpecifier) {
+            // Always use 'node:module'
+            const newModule = "'node:module'";
+            edits.push(moduleSpecifier.replace(newModule));
+        }
```

Comment:
If you use string_fragment, you get only the text inside single or double quotes, without including the quotes themselves. This way, you can replace the string’s content without changing the user’s preference for single or double quotes.

Nit: The newModule variable can be removed for simplicity.
```suggestion
        const moduleSpecifier = statement.find({ rule: { kind: "string_fragment" } });

        if (moduleSpecifier) {
            // Always use 'node:module'
            edits.push(moduleSpecifier.replace('node:module'));
        }
```